hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/RDBStore.java

```diff
@@ -347,13 +347,13 @@ public Collection<ColumnFamily> getColumnFamilies() {
 
   @Override
   public DBUpdatesWrapper getUpdatesSince(long sequenceNumber)
-      throws SequenceNumberNotFoundException {
+      throws IOException {
     return getUpdatesSince(sequenceNumber, Long.MAX_VALUE);
   }
 
   @Override
   public DBUpdatesWrapper getUpdatesSince(long sequenceNumber, long limitCount)
-      throws SequenceNumberNotFoundException {
+      throws IOException {
```

Comment:
dbUpdatesWrapper.setLatestSequenceNumber(db.getLatestSequenceNumber());
<-- this is as used down should be moved within try block, so that dbwrapper status and log can be added.